### ISSUE TITLE
Output wav file not as expected

### ISSUE THREAD

**Issue by taubaaron** [spiyer99/spiyer99.github.io#3]
Hi!, First of all thanks very much for sharing, was very interesting to read.
I tried to recreate the outputs myself but for some reason the wav file that was created was just a bad version (probably reconstructed) of the content audio, with no resemblance of the style audio.
I am running the code in Pycharm and haven't changed any of the actual code besides the hyper parameters. I have tried playing with content/style weights, numsteps and learning rate. None have produced an output with similar style to the style audio. 
Am I doing something wrong?
Do you have examples of outputs that have worked?
TIA!

**Comment by neelriyer** [comment]
Hey @taubaaron, which blog post/notebook were you referring to?

**Comment by taubaaron** [comment]
Hey @spiyer99 Thanks for your reply.
I was reading this article: [https://spiyer99.github.io/Change-Audio-Pytorch/](url)

I tried running it myself with the following hyper parameters:

> Style Weight:10000, Content Weight:2, Epochs:500, Learning Rate:0.0001

Where does the actual style learning happen? Am I missing something?

Also, is it saving the weights of the network? I mean if I wanted to run a "test" could it be done in 1 epoch?

TIA

**Comment by neelriyer** [comment]
So the style learning happens in `CNNModel`. It doesn't save the weights automatically, but you can definitely add that in. To do a "test" you'll probably need more than 1 epoch. 

Tbh though the output on my machine wasn't very good. This was just an experiment. Please go ahead and add your own changes in!

Here's a few links that might help you:
https://github.com/alishdipani/Neural-Style-Transfer-Audio
https://pytorch.org/tutorials/advanced/neural_style_tutorial.html
